api/src/extensions.ts

```diff
@@ -500,7 +499,7 @@ class ExtensionManager {
 				}
 			},
 			embed: (position: 'head' | 'body', code: string | EmbedHandler) => {
-				const content = typeof code === 'function' ? code(env) : code;
+				const content = typeof code === 'function' ? code() : code;
```

Comment:
What's the point of using function now?
Sorry for being PITA

Reply:
Nothing wrong with asking critical questions 😄 it has lost most of it's utility when not being used to passed the env however I do like that this allows you to create a function scope for this hook (similar to the others) so you don't have to do any variable declaration or conditional logic in the root extension scope (or separate named function).
I think for both consistency in hooks and developer experience it still adds something.

Reply:
Fair enough, this makes sense.
Thank you